src/components/forms/forms.js

```diff
@@ -285,29 +279,55 @@
   ]);
 };
 
+const formInputRadio = (config) => {
+  const {
+    id, disabled, radioOptions, formValue
+  } = config;
+
+  return radioOptions.map(({value, displayText}) => {
+    return RadioButton({
+      key: `${id}-${value}`,
+      value,
+      defaultChecked: formValue === value,
+      onClick: (event) => onFormInputChange(config, value),
+      description: displayText,
+      style: { fontFamily: 'Montserrat', color: '#1f3b50'}
+    });  
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=10042654)
```suggestion
    });
```